src/dir.props

```diff
@@ -7,4 +7,8 @@
     <SolutionFile Condition="'$(OsEnvironment)'=='Windows_NT'">$([System.IO.Path]::Combine($(SolutionRoot),"MSBuild.sln"))</SolutionFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EnableDocumentationFile)' == 'true'">
+    <DocumentationFile>$(OutputPath)$(TargetName).xml</DocumentationFile>
```

Comment:
Yeah, the property works for me.

---
In reply to: [78443213](https://github.com/Microsoft/msbuild/pull/1015#discussion_r78443213) [](ancestors = 78443213,78442785)